spring-data-jdbc/src/main/java/org/springframework/data/jdbc/core/convert/DefaultDataAccessStrategy.java

```diff
@@ -545,13 +548,26 @@ private IdentifierProcessing getIdentifierProcessing() {
 	private void addConvertedPropertyValue(SqlIdentifierParameterSource parameterSource,
 			RelationalPersistentProperty property, @Nullable Object value, SqlIdentifier name) {
 
-		addConvertedValue(parameterSource, value, name, converter.getColumnType(property), converter.getSqlType(property));
```

Comment:
`JdbcConverter#getSqlType` will invoke `getColumnType` internaly. This is inefficient, since we already called `converter#getColumnType()` for this `RelationalPersistentProperty`